src/cljs/athens/events/remote.cljs

```diff
@@ -32,6 +32,17 @@
                         [:db/sync]]]]}))
 
 
+(rf/reg-event-fx
+  :remote/connection-failed
+  (fn [_ _]
+    (js/console.log ":remote/connection-failed")
+    {:fx [[:dispatch-n [[:loading/unset]
+                        [:conn-status :disconnected]
+                        [:db/sync]
+                        ;; THIS was not recommendation from Stu
```

Comment:
Bad UX leads to disapproving looks from @shanberg, disapproving looks like to mulling about a better UX, mulling leads to prototypes. Yes yes.

Reply:
Psst @shanberg